### PR TITLE
Modem-config : Add modem-configs files for Kumano, Tama and Ganges platforms

### PR DESCRIPTION
**Pull request by Paulbouchara**

### CODE REVIEW

**Comment by oshmoun** [comment]
why `/vendor/oem/modem-config` and not just `/vendor/modem-config`, or maybe `/vendor/etc/modem-config`?

**Comment by MarijnS95** [comment]
@oshmoun Because `/oem` is hardcoded in the modem-switcher utility. `/vendor/oem` will be bind-mounted there.

File: common.mk [hunk 1]
```diff
@@ -125,6 +125,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_COPY_FILES += \
     $(COMMON_PATH)/rootdir/vendor/etc/apns-conf.xml:system/etc/apns-conf.xml
 
+MODEM_CONFIG := $(shell find $(COMMON_PATH)/rootdir/vendor/oem -type f -printf '%p:$(TARGET_COPY_OUT_VENDOR)/oem/%P\n')
```
Comment (kholk) [inline common.mk:128]: ifeq MODEM_CONFIG_LEGACY_PLATFORM <=== Loire Tone Yoshino Nile
MODEM_CONFIG := $(shell find $(COMMON_PATH)/rootdir/vendor/oem/modem-config-legacy  BLAHBLAHBLAH)
else <==== NOT LEGACY: Ganges Tama Kumano Seine && newer
MODEM_CONFIG := whatever you added already
endif

**Comment by kholk** [comment]
Please add the changes that I've requested in the MK and then merge this thing ASAP.

File: rootdir/vendor/etc/init/hw/init.common.rc [hunk 2]
```diff
@@ -112,6 +112,8 @@ on late-fs
     # cause services for bootanim not running.
     wait_for_prop hwservicemanager.ready true
 
+    mount none /vendor/oem /oem bind rec
```
Comment (ix5) [inline rootdir/vendor/etc/init/hw/init.common.rc:115]: Can you split this into a separate file please?

Comment (kholk) [reply rootdir/vendor/etc/init/hw/init.common.rc:115]: Can we do that later? It's a lil late, the implementation has to go live for release....

**Comment by Paulbouchara** [comment]
The files goes where it should and legacy flag is indeed working as it should @jerpelea

**Comment by kholk** [comment]
Since it's very very late, can we just merge this right now, please?

**Pull request merged**